XLS-20/README.md

```diff
@@ -0,0 +1,828 @@
+
+In `direct` mode, an **`NFTokenOfferAccept`** transaction **MUST** fail if:
+
+1. The **`NFTokenOffer`** against which **`NFTokenOfferAccept`** transaction is placed is an offer to `buy`
+   the **`NFToken`** and the account executing the **`NFTokenOfferAccept`** is not, at the time of execution, the current owner of the corresponding **`NFToken`**.
+2. The **`NFTokenOffer`** against which **`NFTokenOfferAccept`** transaction is placed is an offer to `sell`
+   the **`NFToken`** and was placed by an account which is not, at the time of execution, the current owner of the **`NFToken`**.
+3. The **`NFTokenOffer`** against which **`NFTokenOfferAccept`** transaction is placed is an offer to `sell`
+   the **`NFToken`** and was placed by an account which is not, at the time of execution, the `Account` in the recipient field of the **`NFTokenOffer`**, if one exists.
+4. The **`NFTokenOffer`** against which **`NFTokenOfferAccept`** transaction is placed specifies an `expiration` time and the close time field of the parent of the ledger in which the transaction
+   would be included has already passed.
+5. The **`NFTokenOffer`** against which **`NFTokenOfferAccept`** transaction is placed to buy or sell the **`NFToken`**
+   is owned by the account executing the **`NFTokenOfferAccept`**.
+
+If the transaction is executed successfully then:
+
+1. The relevant **`NFToken`** will change ownership, meaning that the token will be removed from the **`NFTokenPage`** of the existing `owner` and be added to the **`NFTokenPage`** of the new `owner`.
+2. Funds will be transferred from the buyer to the seller, as specified in the **`NFTokenOffer`**. If the corresponding **`NFToken`** offer specifies a `TransferRate`, then the `issuer` receives the specified percentage, with the balance going to the seller of the **`NFToken`**.
+
+##### 1.5.6.2.2. Brokered Mode
+
+In `brokered` mode, **`NFTokenOfferAccept`** transaction **MUST** fail if:
+
+1. The `buy` **`NFTokenOffer`** against which **`NFTokenOfferAccept`** transaction is placed is owned by the account executing the transaction.
+2. The `sell` **`NFTokenOffer`** against which **`NFTokenOfferAccept`** transaction is placed is owned by the account executing the transaction.
+3. The account which placed the offer to sell the **`NFToken`** is not, at the time of execution, the current owner of the corresponding **`NFToken`**.
+4. Either offer (`buy` or `sell`) specifies an `expiration` time and the close time field of the parent of the ledger in which the transaction would be included has already passed.
+5. The `owner` of the `sell` **`NFTokenOffer`** is the same account as the `owner` of the `buy` **`NFTokenOffer`**. In other words, the **`NFToken`** cannot be sold to the account that currently owns it.
+6. The account, that submitted the **`NFTokenOfferAccept`**  transaction, has a different address from what is specified in the `Destination` field of the `sell`/`buy` **`NFTokenOffer`**.
```

Comment:
6. The account that submitted the **`NFTokenOfferAccept`**  transaction has a different address from the address specified in the `Destination` field of the `sell`/`buy` **`NFTokenOffer`**.